lib/beaker/hypervisor/openstack.rb

```diff
@@ -63,6 +63,17 @@ def initialize(openstack_hosts, options)
         raise "Unable to create OpenStack Network instance (api_key: #{@options[:openstack_api_key]}, username: #{@options[:openstack_username]}, auth_url: #{@options[:openstack_auth_url]}, tenant: #{@options[:openstack_tenant]})"
       end
 
+      # Validate openstack_volume_support setting value, reset to boolean if passed via ENV value string
+      if not @options[:openstack_volume_support].is_a?(TrueClass) || @options[:openstack_volume_support].is_a?(FalseClass)
+        if @options[:openstack_volume_support].match(/\btrue\b/i)
+          @options[:openstack_volume_support] = true
+        elsif @options[:openstack_volume_support].match(/\bfalse\b/i)
+          @options[:openstack_volume_support] = false
+        else
+          raise "Invalid value provided for CONFIG setting openstack_volume_support, current value #{@options[:openstack_volume_support]}"
+        end
+      end
```

Comment:
@kevpl The reason for that block (67-75) is to handle inputs from the environment var.  Basically checking if the string is true/false and then converting to an actual boolean.  I was trying to find a way to just cast it as needed, but couldn't get it working and fell back to this.  Open to suggestions :)